### PR TITLE
Fixes the discontinuous lagrange reference coords dispatch

### PR DESCRIPTION
**Pull request by koehlerson**

### CODE REVIEW

File: src/interpolations.jl [hunk 1]
```diff
@@ -153,9 +153,18 @@ function value(ip::DiscontinuousLagrange{dim,shape,order}, i::Int, ξ::Vec{dim})
...
     return [Vec{dim, Float64}(ntuple(x->0.0, dim))]
 end
+
+function reference_coordinates(ip::DiscontinuousLagrange{2,RefTetrahedron,0})
+    return [Vec{2,Float64}((1/3,1/3))]
+end
+
+function reference_coordinates(ip::DiscontinuousLagrange{3,RefTetrahedron,0})
+   return [Vec{3,Float64}((1/4,1/4,1/4))]
+end
```
Comment (termi-official) [inline src/interpolations.jl:166]: We had a general solution via ntuple, right?

**Review by termi-official — CHANGES_REQUESTED** [review]

File: src/interpolations.jl (hunk 1 above)
Comment (koehlerson) [reply src/interpolations.jl:166]: no :D that's 1:1 the piece of code of the other branch

**Review by termi-official — APPROVED** [review]

**Comment by fredrikekre** [comment]
Add a test? :)

**Comment by termi-official** [comment]
I already tried a bit of stuff to test interpolations, but I cannot find a good angle. Do you have something in mind how we can improve interpolation testing @fredrikekre ?

**Pull request merged**